plugins/inputs/influxdb_v2_listener/influxdb_v2_listener.go

```diff
@@ -294,6 +297,13 @@ func (h *InfluxDBV2Listener) handleDefault() http.HandlerFunc {
 	}
 }
 
+func (h *InfluxDBV2Listener) handlePing() http.HandlerFunc {
+	return func(res http.ResponseWriter, _ *http.Request) {
+		defer h.pingServed.Incr(1)
+		res.WriteHeader(http.StatusNoContent)
```

Comment:
The [docs](https://docs.influxdata.com/influxdb/v2/api/v2/#operation/GetPing) say that this endpoint also returns the header `X-Influxdb-Build` and  
`X-Influxdb-Version` when not InfluxDB cloud.

Reply:
Why not return `Telegraf` for `X-Influxdb-Build` and [`internal.FormatFullVersion()`](https://github.com/influxdata/telegraf/blob/65778a3723b3666aea173978a93db628be31d600/internal/internal.go#L52) for `X-Influxdb-Version`?